src/context/__tests__/QuoteContext.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+
+import { QuoteProvider, QuoteContext } from '../QuoteContext';
+
+function renderQuoteContextConsumer(quote, handleQuoteChange) {
```

Comment:
You should provide default parameter for `handleQuoteChange`, since one of your test cases do not define it.